Code/Tools/LuaIDE/Source/LUA/WatchesPanel.cpp

```diff
@@ -7,9 +7,12 @@
  */
 
 #include "WatchesPanel.hxx"
-#include <Source/LUA/moc_WatchesPanel.cpp>
+
+#include "AzCore/Debug/Trace.h"
```

Comment:
Can you make this one use the < > syntax ?

Reply:
Oh no, looks like I missed that.